rmw/include/rmw/rmw.h

```diff
@@ -2031,15 +2058,63 @@ rmw_get_node_names(
   rcutils_string_array_t * node_names,
   rcutils_string_array_t * node_namespaces);
 
-/// Return a list of node name and namespaces discovered via a node with its enclave.
+/// Return all known nodes' names, namespaces, and enclave names.
 /**
- * Similar to \ref rmw_get_node_names, but it also provides the enclave name.
+ * Similar to rmw_get_node_names(), but it also provides enclave names.
```

Comment:
Not really a sentence.  Maybe:

```suggestion
 * This is similar to rmw_get_node_names(), but it also provides enclave names.
```

Reply:
See https://github.com/ros2/rmw/pull/272/commits/cbabbacaa854d3087e67a57788b6eaf1333aed98.